external-import/misp/src/requirements.txt

```diff
@@ -2,3 +2,4 @@ pycti==5.6.2
 urllib3==1.26.7
 pymisp
 python-dateutil==2.8.2
+git+https://github.com/MISP/misp-stix.git
```

Comment:
`misp-stix==2.4.169` should be fine as `misp-stix` is now on PyPi:  https://pypi.org/project/misp-stix/